projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -956,6 +957,41 @@ function ( $site_purchase ) {
 	return empty( $bundle_purchases );
 }
 
+/**
+ * Determines whether or not the Install the mobile app task should be visible.
+ *
+ * @return bool True if the Install the mobile app task should be visible.
+ */
+function wpcom_launchpad_is_mobile_app_installed_visible() {
+	$is_atomic_site = ( new Automattic\Jetpack\Status\Host() )->is_woa_site();
+	// If the site is not an Atomic site, we should not show the task.
```

Comment:
This comment doesn't match the logic.
```suggestion
	// If the site is an Atomic site, we should not show the task.
```